run/era5_topaz4_maker.py

```diff
@@ -65,37 +65,47 @@ def topaz4_source_file_name(field, unix_time):
         return f"TP4DAILY_{unix_tm.tm_year}{unix_tm.tm_mon:02}_3m.nc"
 
 # Returns bilinearly interpolated data given array of fractional indices
+# 2023-03-28 Add a wrap-around for the ERA longitude. This is formally
+# incorrect when this function is used for TOPAZ data, but since the target
+# point would have to be out of bounds of the source, it is not so important.
 def bilinear(eyes, jays, data):
     i = np.floor(eyes).astype(int)
     j = np.floor(jays).astype(int)
     
     fi = eyes - i
     fj = jays - j
 
+    iwrap = (i + 1) % data.shape[1]
+
     return ((1 - fj) * (1 - fi) * data[j, i] +
-        (1 - fj) * (fi) * data[j, i + 1] +
+        (1 - fj) * (fi) * data[j, iwrap] +
         (fj) * (1 - fi) * data[j + 1, i] +
-        (fj) * (fi) * data[j + 1, i + 1])
+        (fj) * (fi) * data[j + 1, iwrap])
     
 # Returns bilinearly interpolated data given array of fractional indices, when some of the data missing
+# 2023-03-28 Add a wrap-around for the ERA longitude. This is formally
+# incorrect when this function is used for TOPAZ data, but since the target
+# point would have to be out of bounds of the source, it is not so important.
 def bilinear_missing(eyes, jays, data, missing):
     i = np.floor(eyes).astype(int)
     j = np.floor(jays).astype(int)
     
     fi = eyes - i
     fj = jays - j
 
+    iwrap = (i + 1) % data.shape[1]
+
     dataplier = data != missing # False is zero
 
     weighted_sum = ((1 - fj) * (1 - fi) * data[j, i] * dataplier[j, i] +
-        (1 - fj) * (fi) * data[j, i + 1] * dataplier[j, i] +
-        (fj) * (1 - fi) * data[j + 1, i] * dataplier[j, i] +
-        (fj) * (fi) * data[j + 1, i + 1] * dataplier[j, i])
+        (1 - fj) * (fi) * data[j, iwrap] * dataplier[j, iwrap] +
+        (fj) * (1 - fi) * data[j + 1, i] * dataplier[j + 1, i] +
```

Comment:
Is the second `j+1` right here? The previous code was `j`.

Reply:
See comment on line 101.